Zend/tests/isset_str_offset.phpt

```diff
@@ -60,7 +60,7 @@ var_dump(isset($str[$f]));
 print "done\n";
 
 ?>
---EXPECT--
+--EXPECTF--
```

Comment:
Not sure about this one. Problem is that the current code just silently ignores all invalid types and returns false for them -- even the completely invalid ones like using an array as an offset. (Unlike the array case where this *does* result in a TypeError.)

Reply:
Though I do think that is *should* error, so I'm inclined to accept this...